recipe/install_dll.bat

```diff
@@ -9,6 +9,4 @@ if %ARCH% == 32 (
 robocopy ..\builds\libcurl-vc%VS_MAJOR:"=%-%ARCH_STRING%-release-dll-zlib-dll-ssh2-dll-ipv6-sspi-winssl\ %LIBRARY_PREFIX% *.* /E
 if %ERRORLEVEL% GTR 3 exit 1
 
-del /f %LIBRARY_BIN%\curl.exe
-
 exit 0
```

Comment:
It looks like [`robocopy` has an `/xf` option]( https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy#file-selection-options ), which would allow us to exclude a file. Maybe we can use that to exclude copying `curl.exe`? IDK if that helps, just one thought :)

Reply:
How is excluding any different from copying and deleting?

Reply:
I wouldn't think it would be any different. Then again I wouldn't have thought copying and deleting would have caused us issues. Maybe comment ( https://github.com/conda-forge/curl-feedstock/pull/83#discussion_r496879079 ) provides more context?

Reply:
Don't know and it probably won't work but I had to try it :smile:

One alternative would to get rid of `install_dll.bat`. It seems relatively easy except for that arch32/64 if-clause. I guess we can drop it b/c we don't build win32 anyway.